bucket/irfanview-lean.json

```
@@ -0,0 +1,81 @@
+{
+    "version": "4.60",
+    "description": "A fast, compact and innovative FREEWARE (for non-commercial use) graphic viewer (without PlugIns)",
```

```suggestion
    "description": "A fast, compact and innovative graphic viewer (without PlugIns)",
```
License information is not needed in description because we have a separate field for it